glog/glog.go

```diff
@@ -35,7 +35,7 @@ func Flush() {
 
 // V is a shim
 func V(level Level) Verbose {
-	return Verbose(bool(klog.V(klog.Level(int32(level)))))
+	return Verbose(klog.V(klog.Level(int32(level))).Enabled())
```

Comment:
so previously Verbose was a bool type, but now it's a struct.
isn't the compiler complaining that we are casting a bool to type Verbose here? :grimacing: 
```
cannot convert Enabled() (type bool) to type Verbose
```
https://play.golang.org/p/JxCC36UsKOn

Reply:
So Verbose has been changed to a struct in klog itself - this is the glog 'shim' which is provided to make transitioning to klog easier.

I've not changed the API surface of the glog package at all, meaning if someone were to use a V level, the message would actually be logged *without* using a klog 'Verbose'. This is not caused by this patch, rather it's how the package was originally written!

Reply:
got it.